.devcontainer/image/devcontainer.json

```diff
@@ -41,4 +42,4 @@
 			"dockerDashComposeVersion": "v2"
 		}
     }
-}
```

Comment:
You should not make any change in any files except the files of the function so please remove this